src/mlpack/methods/ann/layer/elu.hpp

```diff
@@ -90,6 +90,10 @@ namespace ann /** Artificial Neural Network. */ {
  * }
  * @endcode
  *
+ * In the deterministic mode, there is no computation of the derivative.
+ *
+ * @note During training you should set deterministic to false and during
```

Comment:
Perhaps we should adjust the wording here:

``During training deterministic should be set to false and during testing/inference deterministic should be set to true``

Let me know what you think.